django_netjsongraph/tests/base/api.py

```diff
@@ -12,14 +12,22 @@ def receive_url(self):
         t = self.topology_model.objects.first()
         return '/api/receive/{0}/?key=test'.format(t.pk)
 
-    def _set_receive(self, topology_model):
-        t = topology_model.objects.first()
```

Comment:
you could make this a property by prepending `@property` to it, then you can call `self.snapshot_url`